setup/templates/python-services/monitoring-refresh-config/monitoring-refresh-config.py

```diff
@@ -0,0 +1,435 @@
+    new_websites = []
+    removed_websites = []
+    updated_websites = []
+    updated_website_elements = []
+
+    for new_xml_schema in new_xml_root_element.findall('./schemas/schema'):
+        new_xml_website = new_xml_schema.find('website').text
+        old_xml_websites = old_xml_root_element.findall(f"./schemas/schema[website='{new_xml_website}']")
+
+        if len(old_xml_websites) == 0:
+            new_websites.append(new_xml_website)
+        elif xmlhelper.compare_xml_elements(new_xml_schema, old_xml_websites[0]) != 0:
+            updated_websites.append(new_xml_website)
+            updated_website_elements.append(XmlElementDto(new_xml_element=new_xml_schema,
+                                                          old_xml_element=old_xml_websites[0]))
+
+    for old_xml_schema in old_xml_root_element.findall('./schemas/schema'):
+        old_xml_website_attr = old_xml_schema.find('website')
+        if old_xml_website_attr is not None:
+            old_xml_website = old_xml_website_attr.text
+            new_xml_website = new_xml_root_element.findall(f"./schemas/schema[website='{old_xml_website}']")
+
+            if len(new_xml_website) == 0:
+                removed_websites.append(old_xml_website)
+        else:
+            removed_websites.append('<website without name>')
+
+    return ConfigChanges(new_websites=new_websites,
+                         removed_websites=removed_websites,
+                         updated_websites=updated_websites,
+                         updated_website_elements=updated_website_elements)
+
+
+# endregion
+
+# region Fetch functions
+def fetch_required_configs():
+    websites = fetch_websites_for_current_instance()
+    if len(websites.errors) > 0:
+        raise Exception(f'Problem Getting Website List: {websites.errors}')
+    elif len(websites.jsonData) == 0:
+        raise Exception('No Websites found!')
+
+    error_mail_json = fetch_config_attributes('error-email')
+    warning_mail_json = fetch_config_attributes('warning-email')
+    aws_sqs_json = [fetch_config_attributes('aws-sqs')]
+    aws_sqs_trackor_integration_json = [fetch_config_attributes('aws-sqs-trackor-integration')]
+    root_config_json = fetch_config_attributes('monitoring')
+
+    schemas_json = translate_trackor_to_xml_field_names(websites.jsonData, Settings.TRACKOR_TO_XML_TRANSLATION_DICT)
+    trace_json(schemas_json)
+
+    return JsonData(error_mail_json=error_mail_json,
+                    warning_mail_json=warning_mail_json,
+                    aws_sqs_json=aws_sqs_json,
+                    aws_sqs_trackor_integration_json=aws_sqs_trackor_integration_json,
+                    root_config_json=root_config_json,
+                    schemas_json=schemas_json)
+
+
+def fetch_websites_for_current_instance():
+    websites = onevizion.Trackor(
+        trackorType=Settings.TRACKOR_TYPE_WEBSITE,
+        paramToken=Settings.TRACKOR_HOSTNAME
+    )
+    websites.read(
+        filters={
```

Comment:
Add additional filter to prevent collisions with existing centralize monitoring installation:
```

WebSites.read(
	filters={
		'VQS_WEB_ACTIVE':1,
		'WEB_CENTRALIZED_MONITORING':0
		},
```

Check issue comments for details https://trackor.onevizion.com/form/ConfigAppForm.do?id=100107283208&ttid=10009161

Reply:
Done.